src/v2/util/getBreadcrumbPath.ts

```diff
@@ -0,0 +1,34 @@
+import { SerializeMeQueryHook_serializedMe } from '__generated__/SerializeMeQueryHook'
+import { LoadingBreadcrumbChannel } from '__generated__/LoadingBreadcrumbChannel'
+import { LoadingBreadcrumbUser } from '__generated__/LoadingBreadcrumbUser'
+import { LoadingBreadcrumbGroup } from '__generated__/LoadingBreadcrumbGroup'
+
+type Crumbable =
```

Comment:
lol this name

Reply:
vaguely disappointed it wasn't named `type Crumby`

Reply:
I can do that for you